CMSgov/mint-app
# Easi 2610/model plan to user table

# EASI-2610

## Changes and Description

- Model Plan now uses the user table
- DB seed refactored to more easily allow for migration
- Plan Favorite, exposed the user account GQL


## How to test this change
1. run scripts/dev db:seed
   a. ensure data is is seeded as expected
2. Update model plans using the front end and make sure that it is working as expected.



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
